src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
https://ci.appveyor.com/project/LebedevRI/rawspeed/build/66/job/sa02r2rpebugwmg0#L1122
^ that does not fail to compile, and it uses `WIN32`, not `_WIN32`